src/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts

```diff
@@ -212,13 +216,17 @@ class KubeflowTrainingService extends KubernetesTrainingService implements Kuber
                 throw new Error('azureStorageClient is not initialized');
             }
             try {
-                //upload local files to azure storage
+                //upload script files to azure storage
```

Comment:
we are thinking the same thing :). the same here.

Reply:
fixed.